astropy/io/ascii/mrt.py

```diff
@@ -356,9 +356,7 @@ def write_byte_by_byte(self):
                         else:
                             lim_vals = f"[{col.min}/{col.max}]"
                 elif col.fortran_format[0] in ("E", "F"):
-                    lim_vals = (
-                        f"[{floor(col.min * 100) / 100.}/{ceil(col.max * 100) / 100.}]"
-                    )
+                    lim_vals = f"[{floor(col.min * 100) / 100.0}/{ceil(col.max * 100) / 100.0}]"
```

Comment:
As long as I don't need to understand why Ruff is now undoing formatting that was introduced by Black rules in the first place, definitely +1 on this!

Reply:
black's format isn't immutable either: it's updated once a year. Ruff follows a similar schedule and both seems to be introducing identical or very similar changes so far.

Reply:
So it's also changing the default line length again between versions?
Personally I'm all in favour of the longer limits, and the fewer `= ($` constructs, the better, but I thought we had discussed at some length (pun inevitable) that the then-black-default of 88 was preferable to most.

Reply:
[This is described in Ruff documentation](https://docs.astral.sh/ruff/formatter/#line-breaks)